xmcl-keystone-ui/locales/pt-BR.yaml

```diff
@@ -0,0 +1,1727 @@
+  focusLayoutDescription: >-
+    O layout de foco é o layout legado do Minecraft Launcher. Ele foi projetado
+    para ser usado com uma única instância ou poucas instâncias.
+  game:
+    description: >-
+      Escolha um diretório de jogo existente (.minecraft) para importar recursos
+      já existentes. Assim, você pode identificar rapidamente os mods, pacotes
+      de recursos, etc.
+    name: Importar game existente
+  locale:
+    description: >-
+      Selecione o idioma de sua preferência, caso esteja disponível! Se não
+      encontrar seu idioma e quiser ajudar, você pode visitar nosso Github para
+      enviar solicitações de tradução!
+    language: Idioma
+    name: Definir Idioma
+  path: Caminho Atual
+  title: 'Bem-vindo ao X Minecraft Launcher. Antes de começar, precisamos que você '
+shaderPack:
+  deletion: Excluir Shaders Pack
+  deletionHint: Isso excluirá o arquivo do shaders pack em {path} e não poderá ser revertido.
+  disabled: Shader Packs Desabilitados
+  dropHint: Importar Shaders Pack
+  empty: Nenhum shaders pack alocado
+  enable: Usando {name}
+  enabled: Shaders Packs Habilitados
+  manage: Gerenciar Shader Packs
+  name: Shaders Pack
+  noShaderMod: Nenhum Mod de Shader Instalado
+  noShaderModHint: Você não pode usar shaders sem um mod de shaders
+  noShaderModInstallHint: Você precisa instalar um dos seguintes mods para carregar o shaders pack.
+  searchHint: Procurar Shaders Pack
+  selectSearchHint: Procurar e selecionar Shaders Pack
+  showDirectory: Mostrar diretório de shaders packs
+  showFile: Mostrar este shaders pack na pasta
+shared:
+  accept: Aceitar
+  keepSelected: Manter seleção
+  skipForNow: Pular por agora
+store:
+  explore: Explorar
+  latestMinecraft: Última Versão do Minecraft
+  name: Mercado de Modpacks
+  popular: Modpacks Populares
+  recentUpdated: Atualizados Recentemente
+summery: Resumo
+tag:
+  create: Criar uma Tag
+  createSelected: Criar uma Tag para os Selecionados
+  newTag: Nova Tag
+task:
+  available: Disponível
+  cancelled: Cancelada
+  clear: Limpar tarefas finalizadas
+  compatible: Compatível
+  connection:
+    connected: Conectado
+    free: Gratuito
+    pending: Pendente
+    queued: Em fila
+    running: Conexão em andamento
+    size: Contagem total
+  connections: Conexões de Rede
+  defaultDescription: Aparência padrão do Minecraft
+  delete:
+    content: >-
+      Isso excluirá o arquivo texturepack do disco e não será possível
+      recuperá-lo.
+    title: Excluir texturepack
+  dropHint: Arraste e solte as texturas da pasta/zip aqui para importar.
+  empty: Nenhuma Tarefa em Execução
+  enable: 'Pacotes de textura incluídos: {count}'
+  failed: Falhou
+  import: Importar texturepack
+  incompatible: Imcompatível
+  manage: Texturepack
+  manager: Gerenciador de Tarefas
+  nTaskRunning: '{count} tarefas em execução'
+  name: Tarefa | Tarefas
+  pause: Pausar
+  searchOnCurseforge: Pesquisar {nome} no curseforge
```

Comment:
Guess this should be `{name}`?

Reply:
Damn, I'm sorry, that field I translated from the Russian language, the translator probably translated the English part into Portuguese.